src/components/screens/Authenticator/Authenticator.container.vue

```diff
@@ -96,6 +96,10 @@ export default {
       const isEmailOtp = this.challengeType === CHALLENGE_TYPES.EMAIL_OTP;
 
       switch (true) {
+        case name === 'SignIn' && isEmailOtp && !this.isPasswordExist:
+          this.openRoute('SignUp');
```

Comment:
If we have created account, does it correct to force user for sign up? May be we should create another screen with separated functionality?

Reply:
if you doesn't have password, that means, that you are not registered in the system, that's why signUp will showed

Reply:
btw, this is the main goal

Reply:
It is very strange. So, ok